ReactViewResources/Loader/Internal/ComponentWithMountCallback.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
+import { PropsWithChildren, useState, useEffect } from "react";
+
+export interface IComponentWithMountCallbackProps {
+    mounted?(): void;
+}
+
+export function ComponentWithMountCallback({ mounted, children }: PropsWithChildren<IComponentWithMountCallbackProps>): JSX.Element {
+    const [isMounted, setIsMounted] = useState(false);
+    useEffect(() => {
+        setIsMounted(true);
+        mounted?.();
+    }, []);
+    
+    return isMounted ? <>{children}</> : <></>;
```

Comment:
<img width="733" alt="Screenshot 2024-02-16 at 14 57 16" src="https://github.com/OutSystems/ReactView/assets/8090726/fe526433-6b43-4880-8e9f-d5e1ff0f90d3">

Needed this isMounted because I was getting updates after the hydration - which shouldn't happen